newIDE/app/src/ObjectsList/index.js

```diff
@@ -48,15 +33,61 @@ import PreferencesContext from '../MainFrame/Preferences/PreferencesContext';
 import { Column, Line } from '../UI/Grid';
 import ResponsiveRaisedButton from '../UI/ResponsiveRaisedButton';
 import Add from '../UI/CustomSvgIcons/Add';
+import InAppTutorialContext from '../InAppTutorial/InAppTutorialContext';
+import {
+  enumerateFoldersInContainer,
+  enumerateFoldersInFolder,
+  enumerateObjectFolderOrObjects,
+  getObjectFolderOrObjectUnifiedName,
+  type ObjectFolderOrObjectWithContext,
+} from './EnumerateObjectFolderOrObject';
+import { mapFor } from '../Utils/MapFor';
+import IconButton from '../UI/IconButton';
+import AddFolder from '../UI/CustomSvgIcons/AddFolder';
+import { LineStackLayout } from '../UI/Layout';
+import KeyboardShortcuts from '../UI/KeyboardShortcuts';
+import Link from '../UI/Link';
+import { getHelpLink } from '../Utils/HelpLink';
+import useAlertDialog from '../UI/Alert/useAlertDialog';
 
 const gd: libGDevelop = global.gd;
+const sceneObjectsRootFolderId = 'scene-objects';
+const globalObjectsRootFolderId = 'global-objects';
+const globalObjectsEmptyPlaceholderId = 'global-empty-placeholder';
+
+const globalObjectsWikiLink = getHelpLink(
+  '/interface/scene-editor/global-objects/',
+  ':~:text=Global%20objects%20are%20objects%20which,are%20usable%20by%20all%20Scenes'
```

Comment:
is this link too precise? If we change the wording of the doc, this will break. Should we link to a section title instead?

Reply:
The page is quite short so if the copy changes and the browser can't match the anchor, the impact won't be important. I would say let's keep it as is